src/core/execute.c

```diff
@@ -1940,6 +1940,14 @@ static int build_environment(
                 if (!joined)
                         return -ENOMEM;
 
+                if (!strv_isempty(c->directories[t].symlinks)) {
+                        char **src, **dst;
+
+                        STRV_FOREACH_PAIR(src, dst, c->directories[t].symlinks)
+                                if (!strextend_with_separator(&joined, ":", pre, *dst, NULL))
+                                        return -ENOMEM;
+                }
```

Comment:
hmm, actually, thinking about this, should we really list the aliases in the env var? the env var is for finding your data, but the aliases are not helpful, they don't really help you finding anything you couldn't find via the en var already...

Reply:
removed